src/js/components/Diagram/Diagram.js

```diff
@@ -39,10 +45,22 @@ const COMMANDS = {
     cmds += `T ${toPoint[0] + offset},${toPoint[1] + offset}`;
```

Comment:
https://github.com/grommet/grommet/pull/5950#discussion_r842227385
How do we know which anchor should have priority ? The initial `anchor` or `anchor` defined inside the `fromTarget` and `toTarget` ?